src/EA.Weee.Core.Tests.Unit/AatfReturn/AddressDataTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace EA.Weee.Core.Tests.Unit.AatfReturn
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Core.AatfReturn;
+    using FluentAssertions;
+    using Xunit;
+
+    public class AddressDataTests
+    {
+        [Theory]
+        [InlineData("Name")]
```

Comment:
Can also add tests for the max length attribute

Reply:
Added